src/lang/po/Prusa-Firmware-Buddy.pot

```diff
@@ -18,7 +18,7 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 
 #. c=20 r=4
-#: src/gui/screen_menu_version_info.cpp:86
+#: src/gui/screen_menu_version_info.cpp:82
 #, c-format
 msgid ""
 "\n"
```

Comment:
msgid "Bootloader Version"
msgid "Buddy Board"

Reply:
No way! Here the '\n' are important